benchmark/index.js

```diff
@@ -0,0 +1,189 @@
+    const dest = destinationsList[idx];
+    nativeDestHandlers[dest] = versionedRouter.getDestHandler("v0", dest);
+    destCdKWorkflowEngines[dest] = await cdkV2Handler.getWorkflowEngine(
+      dest,
+      TRANSFORMER_METRIC.ERROR_AT.PROC
+    );
+  }
+}
+
+async function run() {
+  await initializeHandlers();
+
+  for (const dest in destTestSet) {
+    for (const td in destTestSet[dest]) {
+      const curTcData = destTestSet[dest][td];
+      let tcInput;
+      let tcDesc;
+      if (
+        "description" in curTcData &&
+        "input" in curTcData &&
+        "output" in curTcData
+      ) {
+        tcInput = curTcData.input;
+        tcDesc = `Destination - ${dest} - "${curTcData.description}"`;
+      } else {
+        tcInput = curTcData;
+        tcDesc = `Destination - ${dest} - "${td}"`;
+      }
+
+      await runDataset(tcDesc, tcInput, dest);
+    }
+  }
+
+  for (const src in srcTestSet) {
+    for (const td in srcTestSet[src]) {
+      const curTcData = srcTestSet[src][td];
+      let tcInput;
+      let tcDesc;
+      if (
+        "description" in curTcData &&
+        "input" in curTcData &&
+        "output" in curTcData
+      ) {
+        tcInput = curTcData.input;
+        tcDesc = `Source - ${src} - "${curTcData.description}"`;
+      } else {
+        tcInput = curTcData;
+        tcDesc = `Source - ${src} - "${td}"`;
+      }
+
+      await runDataset(tcDesc, tcInput, src);
+    }
+  }
+}
+
+async function runDataset(desc, input, intg) {
+  const suiteName = desc;
+  logger.info("==========================================");
+  logger.info(suiteName);
+  logger.info("==========================================");
+  const results = {};
+  const suite = new Benchmark(suiteName, "Operations");
+
+  suite.add("native", async function() {
+    try {
+      await versionedRouter.handleV0Destination(
+        nativeDestHandlers[intg].process,
+        intg,
+        input,
+        TRANSFORMER_METRIC.ERROR_AT.PROC
+      );
+    } catch (err) {
+      // logger.info(err);
+      // Do nothing
+    }
+  });
+
+  suite.add("CDK 2.0", async function() {
+    try {
+      await cdkV2Handler.process(destCdKWorkflowEngines[intg], input);
+    } catch (err) {
+      // logger.info(err);
+      // Do nothing
+    }
+  });
+
+  suite
+    .on("cycle", function(result) {
+      results[result.end.name] = { stats: result.end.stats };
+    })
+    .on("complete", function(result) {
+      logger.info(
+        "Fastest: ",
+        result.end.name,
+        result.end.stats.n,
+        result.end.stats.mean.toFixed(4)
+      );
+      logger.info();
+      Object.keys(results).forEach(impl => {
+        logger.info(
+          impl,
+          results[impl].stats.n,
+          results[impl].stats.mean.toFixed(4)
+        );
+        logger.info(
+          `"${result.end.name}" is faster by ${(
+            result.end.stats.n / results[impl].stats.n
+          ).toFixed(1)} times to "${impl}"`
+        );
```

Comment:
```suggestion
       if(result.end.name !==  results[impl].name){
          logger.info(
            `"${result.end.name}" is faster by ${(
              result.end.stats.n / results[impl].stats.n
            ).toFixed(1)} times to "${impl}"`
          );
        }
```

Reply:
Fixed this as part of the latest updates.